rocky/rocky/templates/partials/list_filters.html

```diff
@@ -4,12 +4,21 @@
 <section class="filter">
     <div>
         <p>
-            {% block title %}{% endblock title %}
+            {% block title %}
+            {% endblock title %}
         </p>
         <button aria-expanded="false"
                 data-hide-filters-label="{% block hide_text %}{% translate "Hide filter options" %}{% endblock hide_text %} ">
             {% translate "Show filter options" %}
         </button>
+        {% if request.GET %}
```

Comment:
This seems like a wide net to throw out? any get request means we have filters one?

Reply:
I will make it more specific